docs/src/Tooltip.doc.js

```diff
@@ -1,165 +1,822 @@
+      bgColor="white"
+      icon="text-align-left"
+      iconColor="darkGray"
+      size="lg"
+    />
+  </Tooltip>
+  <Tooltip text="Align center">
+    <IconButton
+      accessibilityLabel=""
+      bgColor="white"
+      icon="text-align-center"
+      iconColor="darkGray"
+      size="lg"
+    />
+  </Tooltip>
+  <Tooltip text="Align right">
+    <IconButton
+      accessibilityLabel="Align right"
+      bgColor="white"
+      icon="text-align-right"
+      iconColor="darkGray"
+      size="lg"
+    />
+  </Tooltip>
+</Flex>`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        type="don't"
+        description="Use Tooltip to communicate critical information, such as an error, instructions for performing a task, or interaction feedback."
+        defaultCode={`
+<Tooltip text="Pssst! Looks like you've already saved this Pin.">
+  <Button color="red" text="Save" inline size="lg" />
+</Tooltip>
+`}
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection>
+      <MainSection.Card
+        cardSize="md"
+        type="do"
+        description="Use Tooltip to add supplementary information about a feature, typically paired with an `info-circle` [IconButton](/IconButton)."
+        defaultCode={`
+<Tooltip inline text="Total ad spend in the select time period">
+  <Icon
+    accessibilityLabel="Additional info."
+    icon="info-circle"
+    inline
+    color="gray"
+    size={16}
+  />
+  </Tooltip>
+`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        type="don't"
+        description="Pair a Tooltip with a disabled element. See [disabled items](#Disabled-itemds) to learn more."
+        defaultCode={`
 <Tooltip
-  inline
-  text="Logout"
   link={
-    <Link href="https://pinterest.com" target="blank">
-      <Text color="white" size="sm" weight="bold">
+    <Text color="white" size="sm" weight="bold">
+      <Link
+        href="https://help.pinterest.com/en/business/article/get-a-business-account"
+        target="blank"
+      >
         Learn more
-      </Text>
-    </Link>
+      </Link>
+    </Text>
   }
+  text="There was a problem converting to a personal account."
 >
-  <Icon
-    accessibilityLabel="Logout of your profile"
-    color="pine"
-    icon="logout"
+  <Button size="lg" disabled text="Convert to personal account" />
+</Tooltip>
+`}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
+);
+
+card(
+  <MainSection
+    name="Accessibility"
+    description={`
+When using Tooltip with [IconButton](/IconButton), avoid repetitive labeling. The accessibilityLabel provided to IconButton should describe the intent of the button, not the icon itself. For instance, use “Settings” instead of “Cog icon”. Tooltip text can expand upon that intention, as seen with the cog IconButton. If the Tooltip text and IconButton accessibilityLabel contain the same content, pass an empty string to \`accessibilityLabel\`, as seen with the send IconButton.`}
+  >
+    <MainSection.Subsection>
```

Comment:
That description, and a `title="Labels"` should be on this Subsection, not the MainSection

Reply:
DONE